data/metadataOverwrite.json

```
@@ -25,6 +25,11 @@
 			"decimals": 9,
 			"symbol": "DGD",
 			"name": "DigixDAO DGD Token"
+		},
+		"0x960b236A07cf122663c4303350609A66A7B288C0": {
+			"decimals": 18,
+			"symbol": "ANT (old)",
```

If this appear on the UI i would suggest something like: 
```suggestion
			"symbol": "ANT v1",
```

---

Works for me, but they suggested "(old)" suffix.

---

Ok then we can keep old

---

Hey guys :). Yes, for token symbols we're trying to align with "ANT (old)" (ANTv1) and "ANT" (ANTv2) to make it less confusing for users, especially as v1 will disappear over time.